package.json

```diff
@@ -60,22 +60,20 @@
     "@types/react": "^18.0.3",
     "@types/react-dom": "^18.0.0",
     "@types/react-test-renderer": "^18.0.0",
-    "esbuild": "^0.21.3",
-    "jest": "^27.5.1",
-    "jest-environment-jsdom": "^27.4.4",
+"esbuild": "^0.21.3",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
     "jest-environment-jsdom-global": "^4.0.0",
-    "jest-junit": "^16.0.0",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-test-renderer": "^18.0.0",
-    "rimraf": "^5.0.7",
+    "rimraf": "^5.0.9",
     "rollup": "^3.26.2",
     "rollup-plugin-dts": "^6.1.0",
     "rollup-plugin-esbuild": "^6.1.1",
-    "rollup-plugin-filesize": "^10.0.0",
```

Comment:
Removing this plugin. We may want to add a similar plugin in the future, but I am working on minimizing requirements that are not strictly needed at the moment.